cmd/template/echoRoutes.go

```
@@ -43,5 +48,43 @@ func (s *Server) helloWorldHandler(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, resp)
 }
+
+func (s *Server) pingPongWebsocketHandler(c echo.Context) error {
+	w := c.Response().Writer
+	r := c.Request()
+	errorMessage := []byte("This is another message not PING")
+	socket, err := websocket.Accept(w, r, nil)
+
+	if err != nil {
+		log.Print("could not open websocket")
```

We're eating the error here and not printing it anywhere